projects/neoforge/build.gradle

```diff
@@ -222,6 +222,11 @@ generateAccessTransformers {
     )
 }
 
+tasks.withType(JavaCompile.class).configureEach {
```

Comment:
Do we need the same in the base project?

Reply:
Running with `./gradlew :base:compileJava :base:assemble --no-build-cache --rerun-tasks` with the same setup that causes the OOM error with `:neoforge:compileJava` does not cause an error, so it seems that the base subproject does not need the fix. That is also corroborated with the logs from my failed builds, which all run `:base:compileJava` successfully then fail on `:neoforge:compileJava`.

If wanted though as future-proofing, we could copy it into the base subproject or move it somewhere to be applied to both subprojects.

Reply:
Good enough as is then. If it happens to `base` in the future, we know how to fix it.